packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/hooks/useReviewWorkflows.js

```diff
@@ -28,22 +30,52 @@ export function useReviewWorkflows(workflowId) {
     return data;
   }
 
+  async function updateEntityStage({ entityId, stageId }) {
+    const {
+      data: { data },
+      // TODO: endpoint differs for collection-types and single-types
+    } = await put(`${API_CM_BASE_URL}/collection-types/:model_uid/${entityId}/stage`, {
+      data: { id: stageId },
+    });
+
+    return data;
+  }
+
   function updateWorkflowStages(workflowId, stages) {
     return workflowUpdateMutation.mutateAsync({ workflowId, stages });
   }
 
+  function setStageForEntity(entityId, stageId) {
+    return entityStageMutation.mutateAsync({ entityId, stageId });
+  }
+
   function refetchWorkflow() {
     client.refetchQueries(workflowQueryKey);
   }
```

Comment:
Are you sure these don't need to be used in effects and therefore would benefit from being wrapped in `useCallback`? 🤔  just an API thought.

Reply:
No, they don't have to. The can be called in a hook of the component that is consuming it though.

Reply:
> The can be called in a hook of the component that is consuming it though.

This is what I meant sorry, so if I use the hook somewhere else... But maybe that should be done on a case by case basis?

Reply:
Yes ... I've made a mistake and shouldn't have included that in this PR already. There will be a separate one just for updating :)